portals/publisher/src/main/webapp/site/public/locales/en.json

```diff
@@ -9641,6 +10099,24 @@
       "value": "The server encountered an internal error or misconfiguration and was unable to complete your request."
     }
   ],
+  "globalPolicies.create.create.heading": [
+    {
+      "type": 0,
+      "value": "Create A New Global Policy"
+    }
+  ],
+  "globalPolicies.create.edit.heading": [
+    {
+      "type": 0,
+      "value": "Edit Global Policy"
```

Comment:
```suggestion
      "value": "Edit global policy"
```

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)